src/mono/netcore/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -277,14 +277,20 @@
       <Compile Include="$(BclSourcesRoot)\System\IO\MonoIOError.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
+      <Compile Include="$(BclSourcesRoot)\System\ApplicationModel.Windows.cs" />
```

Comment:
I do not think you need `ApplicationModel.Windows.cs` now that FeatureUap is not set.

Reply:
Good catch. Removed.